appium/webdriver/webdriver.py

```diff
@@ -341,51 +332,34 @@ def find_elements_by_android_uiautomator(self, uia_string):
         """
         return self.find_elements(by=By.ANDROID_UIAUTOMATOR, value=uia_string)
 
-    def find_element_by_image(self, png_img_path,
-                              match_threshold=DEFAULT_MATCH_THRESHOLD):
+    def find_element_by_image(self, png_img_path):
         """Finds a portion of a screenshot by an image.
         Uses driver.find_image_occurrence under the hood.
 
         :Args:
         - png_img_path - a string corresponding to the path of a PNG image
-        - match_threshold - a double between 0 and 1 below which matches will
-          be rejected as element not found
 
-        :return: an ImageElement object
+        :return: an Element object
         """
-        screenshot = self.get_screenshot_as_base64()
         with open(png_img_path, 'rb') as png_file:
             b64_data = base64.b64encode(png_file.read()).decode('UTF-8')
-        try:
-            res = self.find_image_occurrence(screenshot, b64_data,
-                                             threshold=match_threshold)
-        except WebDriverException as e:
-            if 'Cannot find any occurrences' in str(e):
-                raise NoSuchElementException(e)
-            raise
-        rect = res['rect']
-        return ImageElement(self, rect['x'], rect['y'], rect['width'],
-                            rect['height'])
-
-    def find_elements_by_image(self, png_img_path,
-                               match_threshold=DEFAULT_MATCH_THRESHOLD):
+
+        return self.find_element(by=By.IMAGE, value=b64_data)
+
+    def find_elements_by_image(self, png_img_path):
```

Comment:
it is no necessarily a png file. The format might be anything support by OpenCV itself

Reply:
💡